vector/src/main/res/drawable/ic_splash_collaboration.xml

```diff
@@ -0,0 +1,1368 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="300dp"
+    android:height="300dp"
+    android:viewportWidth="300"
+    android:viewportHeight="300">
+  <group>
+    <clip-path
```

Comment:
pretty sure lint is already complaining about the complexity of those vectors...
Not a strong opinion neither, but probably quite bad for the performance than classical png/webp images

Reply:
as these are one time use assets and they're quite complex I'll covert to .png

Reply:
https://github.com/vector-im/element-android/pull/4880/commits/a35dc7f99906c3d76e75bdef796f93fbbd64cbc0